### PR TITLE
Feat/fro 105/heed dashboard 5

### PR DESCRIPTION
**Pull request by VictoriaAde**

### CODE REVIEW

File: scrybe_frontend/src/pages/UploadRecordings/index.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import styles from "./uploadedRecordings.module.scss";
 
 function UploadedRecordings() {
   const [toggleSidebar, setToggleSidebar] = React.useState(false);
-  const [isSearching, setIsSearching] = React.useState("");
+  const [setIsSearching] = React.useState("");
   const setterFn = (e) => {
```
Comment (dprincecoder) [inline scrybe_frontend/src/pages/UploadRecordings/index.jsx:10]: Why did you remove code from a folder, not yours?

**Review by dprincecoder — CHANGES_REQUESTED** [review]

File: scrybe_frontend/src/pages/UploadRecordings/index.jsx (hunk 1 above)
Comment (VictoriaAde) [reply scrybe_frontend/src/pages/UploadRecordings/index.jsx:10]: Oh, sorry. It was an honest mistake. I would push again now @dprincecoder

**Review by dprincecoder — APPROVED** [review]
